gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.9-bin.zip
+#Fri Oct 26 21:08:02 PDT 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-zipStorePath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.9-all.zip
```

Comment:
More IntelliJ -- I believe it made this change when I imported the project. Can be removed if there is a reason we only wanted to use bin.

Reply:
@neoancient thoughts on this?

Reply:
The runtime is sufficient to the task. It won't hurt anuthing, but I see no need to download and save sample code and documentation.

Reply:
Updated to use the runtime (bin) only